middlewares/index.js

```diff
@@ -1,4 +1,5 @@
 export { default as errorHandler } from './errorHandler';
 export { default as userValidations } from './userValidations';
+export { default as profileValidation } from './profileValidation';
```

Comment:
Please pluralize this. I suspect there's not only one validation middleware in that file. It's more semantically correct as `profileValidations`

Reply:
ok... will do